interface/main/finder/dynamic_finder.php

```diff
@@ -60,6 +60,10 @@
 <head>
     <?php Header::setupHeader(['datatables', 'datatables-colreorder', 'datatables-dt']); ?>
     <title><?php echo xlt("Patient Finder"); ?></title>
+
+
+    <link rel="stylesheet" href="./../../../public/assets/datatables.net-bs4/css/dataTables.bootstrap4.min.css">
+    <script src="./../../../public/assets/datatables.net-bs4/js/dataTables.bootstrap4.min.js"></script>
```

Comment:
remove above 2 lines and instead change line 61 to:
```
<?php Header::setupHeader(['datatables', 'datatables-colreorder', 'datatables-dt', 'datatables-bs']); ?>
```
In regards to that line, here is where the magic is happening:
https://github.com/openemr/openemr/blob/master/config/config.yaml#L111-L114